Chore(docs) remove section of Reviewing docs to  be moved into PR guide (#11)
mdn/project-template: REVIEWING.md

# Reviewing guide

## Values and guidelines

> All reviewers must abide by the [Code of Conduct](CODE_OF_CONDUCT.md) and are also protected by it.

Reviewer: ```suggestion
All reviewers must abide by the [Code of Conduct](CODE_OF_CONDUCT.md); they are also protected by the Code of Conduct.
```

> A reviewer should not tolerate poor behavior and is encouraged to [report any behavior](CODE_OF_CONDUCT.md#How_to_Report) that violates the Code of Conduct.

Reviewer: This link might change based on my title suggestion in the other file.

How about extending the link to:
```suggestion
A reviewer should not tolerate poor behavior and is encouraged to [report any behavior that violates](CODE_OF_CONDUCT.md#How_to_Report) the Code of Conduct.
```

Author: Good, have updated to match the new anchor 👍🏻

Everyone participating must follow our [Community Participation Guidelines](CODE_OF_CONDUCT.md#Community_Participation_Guidelines)

> ## Code review process

Reviewer: ```suggestion
## Review process
```
..to not be taken literally as code 🙂

> MDN Web Docs have a well-defined review process which must be followed by reviewers in all repositories under this organization.

Reviewer: ```suggestion
The MDN Web Docs team has a well-defined review process that must be followed by reviewers in all repositories under the GitHub MDN organization.
```
makes sense?

> This process is described in the [pull requests](https://developer.mozilla.org/en-US/docs/MDN/Community/Pull_requests) documentation page.

Reviewer: ```suggestion
This process is described in detail on the [Pull request guidelines](https://developer.mozilla.org/en-US/docs/MDN/Community/Pull_requests) page.
```
